src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystemWithPIDForSpeed;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import com.spikes2212.dashboard.Namespace;
+import com.spikes2212.util.TalonEncoder;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+
+import java.util.function.Supplier;
+
+/**
+ * A shooter subsystem that is controlled with two {@link WPI_TalonSRX}.<br>
+ * In addition, it has settings for PID and matching commands, which you can use and control through the {@link Shuffleboard}.
+ */
+public class Shooter extends MotoredGenericSubsystem {
+
+    /**
+     * The distance the wheel moves each encoder's pulse.
```

Comment:
the wheel doesn't move anywhere,
what is a `distance`

Reply:
https://letmegooglethat.com/?q=what+is+a+distance